app/controllers/skills_controller.rb

```diff
@@ -1,49 +1,46 @@
 class SkillsController < ApplicationController
   before_action :signed_in_user
+  respond_to :html, :js
 
   def index
     fetch
   end
 
-  def fetch
-    @skills = Skill.paginate(page: params[:page], per_page: 15 )
-    @page = params[:page]
-  end
-
   def new
     @skill = Skill.new
-    @page = params[:page]
```

Comment:
This code covers scenario:
- user on page 7
- add/remove skill
- index is reloaded, but paginator stays on page 7, but table shows page 1.
Please return the code it handles back.